dali/operators/decoder/audio/audio_decoder_op.cc

```diff
@@ -22,21 +23,85 @@ DALI_SCHEMA(AudioDecoder)
 This operator is a generic way of handling encoded data in DALI.
 It supports most of well-known audio formats (wav, flac, ogg).
 
-This operator produces two outputs:
-output[0]: batch of decoded data
+This operator produces two outputs: <br/>
+output[0]: batch of decoded data <br/>
 output[1]: batch of sampling rates [Hz]
 
 Sample rate (output[1]) at index `i` corresponds to sample (output[0]) at index `i`.
 On the event more metadata will appear, we reserve a right to change this behaviour.)code")
                 .NumInput(1)
                 .NumOutput(detail::kNumOutputs)
+                .AddOptionalArg(detail::kSampleRateName,
```

Comment:
Add some options for resampling quality - we can choose number of sinc lobes based on that.